auto_causality/scoring.py

```diff
@@ -44,6 +46,22 @@ def const_marginal_effect(self, X):
 #         cate_estimate > 0,
 #     )
 #     return erupt_score
+def energy_distance_scores(
+    estimate: CausalEstimate,
+    df: pd.DataFrame,
+):
+    est = estimate.estimator
+    treatment_name = est._treatment_name[0]
+    dy = estimate.estimator.effect(df)
```

Comment:
Almost perfect! However the 'effect' as I understand is the conditional effect if the treatment was applied. However what we need here is the _actual_ effect, so need to set it to zero where treatment == 0